pkg/postrender/exec_test.go

```diff
@@ -116,7 +116,7 @@ func TestExecRun(t *testing.T) {
 	testpath, cleanup := setupTestingScript(t)
 	defer cleanup()
 
-	renderer, err := NewExec(testpath)
+	renderer, err := NewExec(testpath + " BARTEST")
```

Comment:
Could you make a separate test for arguments instead of changing the existing one?  I think it is important to keep a test that makes sure the "no-argument" case still works.

Reply:
Done 👍